pgmax/factor/logical.py

```diff
@@ -176,7 +176,7 @@ def compile_wiring(
         # Note: all the variables in a LogicalFactorGroup are binary
         num_states_cumsum = np.arange(0, 2 * var_states.shape[0] + 2, 2)
         var_states_for_edges = np.empty(shape=(2 * var_states.shape[0],), dtype=int)
-        enumeration._compile_var_states_numba(
+        enum._compile_var_states_numba(
```

Comment:
`logical` and `enum` should be parallel, so `logical` shouldn't be calling functions from `enum`. Move `_compile_var_states_numba` to `factor.py`?